project/Common.scala

```diff
@@ -37,19 +37,7 @@ object Common extends AutoPlugin {
     crossVersion := CrossVersion.binary,
     crossScalaVersions := Dependencies.ScalaVersions,
     scalaVersion := Dependencies.Scala213,
-    scalacOptions ++= Seq(
-        "-unchecked",
-        "-deprecation",
-        "-language:_",
-        //"-Xfatal-warnings", //FIXME need silencer first
-        "-Ywarn-unused",
-        "-encoding",
-        "UTF-8",
-        "-unchecked",
-        "-Xlint",
-        "-Ywarn-dead-code"),
```

Comment:
below we are removing the Xfatal-warnings, what is it that we want to have? Isn't this also related to the Scala 2.12 ticket?

```
--= Seq("-Xfatal-warnings")
```

Reply:
Did you forget to commit the AkkaDisciplinePlugin?

Reply:
yes, I forgot to commit it. Pushed now